geps/gep-713/index.md

```diff
@@ -3,1155 +3,1112 @@
+* a way to augment the behavior of the `Gateway` object itself (e.g. reconcile cloud infrastructure provider settings from the spec declared by the `Gateway` according the rules specified by the metaresource attached to the `Gateway`) or
+* a means to augment the behavior of all `HTTPRoute` objects attached to the `Gateway` (in a way that every new `HTTPRoute` that gets created or modified so it enters the context of the `Gateway` is automatically put in the scope of the metaresource.)
 
-**Ana**: _Huh, I actually don’t have permissions for RetryPolicy._ 🤔
+#### Declared targets versus Effective targets
 
-**Chihiro**: 🤷 _Feels like you should but OK, guess that can’t be it._
+The target kinds specified in the `targetRefs` stanza of a metaresource are referred to as *Declared target* kinds.
 
-Minutes pass while both look at logs.
+These are distinct from *Effective target* kinds, which are the kinds of target objects whose behaviors are actually augmented by the metaresource. That occurs when declared targets are not equal to the actual targets augmented by the metaresource, but rather serve as a means for reaching other levels (typically lower level) of a hierarchy of related object kinds ("hierarchy of nested contexts").
 
-**Chihiro**: _I’m an idiot. There’s a RetryPolicy for the whole namespace –
-sorry, too many policies in the dashboard and I missed it. Deleting that since
-you don’t want retries._
+To avoid ambiguity in the interpretation of the targets, metaresource designs MUST clearly define the extent of the effects of the metaresource respectively to the object kinds they can target (semantics of scoping a metaresource). This can be done via documentation and it typically refers to a known hierarchy of resource kinds.
 
-**Ana**: _Are you sure that’s a good–_
+### Conflicting specs, Inheritance, Merge strategies, and Effective metaresources
 
-Ana’s phone shrills while she’s typing, and she drops it. When she picks it
-up again she sees a stack of alerts. She goes pale as she quickly flips
-through them: there’s one for every single service in the `baker` namespace.
+Declaring additional specifications to objects from the outside can yield conflicts that need to be addressed in the implementation of metaresources. Multiple instances of a metaresource kind may affect an object (directly or indirectly), thus posing a possible conflict to resolve regarding which intent among the multiple metaresource specs a controller shall honor.
 
-**Ana**: _PUT IT BACK!!_
+In some cases, the most recent between two conflicting specs may be desired to win, whereas in other cases it might be the oldest. Often, the winning spec is determined by the hierarchical level within the scope the metaresource applies, and sometimes other criteria must be adopted to resolve conflicts between metaresources ultimately affecting a same target.
```

Comment:
```suggestion
In some cases, the most recent between two conflicting specs may be desired to win, whereas in other cases it might be the oldest. Often, the winning spec is determined by the hierarchical level within the scope the metaresource applies, and sometimes other criteria must be adopted to resolve conflicts between metaresources ultimately affecting the same target.
```